content/hi/docs/reference/glossary/kube-controller-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: क्यूब-कंट्रोलर-मैनजर (kube-controller-manager)
```

Comment:
put kube-controller-manager in aka.

Reply:
Hi @utkarsh-singh1 I didn't get this ? Can you elaborate more? How inside "aka" ?

Reply:
There is `aka:` key below, where you can put alternate translation word in the title.